spec/predicates/human-review-crev.md

```diff
@@ -0,0 +1,161 @@
+# Predicate type: Dependency Reviews (crev)
+
+Type URI: (tentative) https://in-toto.io/attestation/human-review/crev/v0.1
+
+Note: At the time of writing, the crev project uses a separate versioning
+scheme, with the current version being -1. In future, this predicate may adopt
+a URI associated with that project.
+
+Version: v0.1.0
+
+## Purpose
+
+This attestation type is used to describe the results of human review of
+dependency source code. The format is based on the
+[crev project](https://github.com/crev-dev/crev).
+
+## Use Cases
+
+As noted above, crev enables social review of popular open source software
+dependencies. A crev review includes information such as the thoroughness of
+the review, understanding of the source code, and a final rating.
+
+## Model
+
+Most modern software have external dependencies. Dependency review is the
+process of reviewing and verifying the source code of a particular
+dependency, and can be performed by one or more of several actors in the supply
+chain. The developer importing a new dependency can perform the review or a
+dedicated security team can be tasked with it.
+
+## Schema
+
+```json
+{
+    "_type": "https://in-toto.io/Statement/v1",
+    "subject": [{...}],
+    "predicateType": "https://in-toto.io/attestation/human-review/crev/v0.1",
+    "predicate": {
+        "result": "positive|negative",
+        "reviewer": {
+            "idType": "crev",
+            "id": "<ID>",
+            "url": "<URL>"        
+        },
+        "reviewTime": "<TIMESTAMP>",
+        "thoroughness": "high|medium|low",
+        "understanding": "high|medium|low",
+        "comment": "<STRING>"
+    }
+}
+```
+
+### Parsing Rules
+
+This predicate follows the
+[in-toto Attestation Framework's parsing rules](../v1/README.md#parsing-rules).
+
+### Fields
+
+The subject of this predicate type is a specific package and its version in some
+ecosystem.
+
+`result`, _enum_, _required_
+
+Specifies if the overall rating of the dependency is `positive` or `negative`.
+
+`reviewer` _object_, _required_
+
+Identifies the reviewer. This has some meaning for crev's trust proliferation
+aspects, but the identity of the reviewer can also be mapped based on in-toto's
+functionary handling. `idType` is used to determine the contents of `reviewer`.
+The `url` is a reference to the reviewer's crev-proofs repository.
+
+`reviewTime` _Timestamp_, _required_
+
+Indicates time of review creation. `timestamp` in the original crev
+specification.
+
+`thoroughness` _enum_, _required_
+
+Describes how thorough the reviewer was. Must be set to one of `low`, `medium`,
+or `high`.
+
+`understanding` _enum_, _required_
+
+Describes the reviewer's understanding of the dependency code. Must be set to
+one of `low`, `medium`, or `high`.
```

Comment:
As I mentioned in my comment above, I'd suggest we support using all possible values for these fields according to the [cargo-crev](https://github.com/crev-dev/cargo-crev/blob/master/crev-lib/rc/doc/editing-code-review.md#data-fields) data fields spec.